### PR TITLE
[Feature] Internaltional homepage

### PR DESCRIPTION
**Pull request by Lily2point0**
- hide sections to mirror internal homepage
- amend data structure
- add check for locale variations (EU, APAC, US)
- remove duplicates from regional news section to mimic existing checks

### CODE REVIEW

File: bin/lib/homepage.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+			if(edition === 'international' && i === 0) {
+				Utils.saveBase(sectionData.items);
+			}
+			
+			const sectionImages = await getImagesFor(sectionData.items, structure.getPositions(layout));
+			allSectionImages = allSectionImages.concat(sectionImages);
+
+			if(i === 0 && layout === 'landscape') {
+				//Special accomodation for when landscape piece is opinion
```
Comment (railsagainstignorance) [inline bin/lib/homepage.js]: accommodation. Just saying.

File: bin/lib/homepage.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
+if (process.env.NODE_ENV !== 'production') require('dotenv').config();
+const fetch = require('node-fetch');
+const { sections } = require('./page-structure');
+const structure = require('./page-structure');
+const Utils = require('./utils');
```
Comment (railsagainstignorance) [inline bin/lib/homepage.js]: ok, took me a while to get this.
Why not 
const structure = require('./page-structure');
const sections = structure.sections;

File: bin/lib/homepage.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+			.then(res => res.json())
+			.then(data => { return data.annotations })
+			.catch(err => { throw err });
+}
+
+async function getHeadshot(url) {
+	console.log('getHeadshot::', url);
+	return fetch(`${url}?apiKey=${process.env.FT_API_KEY}`)
+			.then(res => res.json())
+			.then(data => { return data })
```
Comment (railsagainstignorance) [inline bin/lib/homepage.js]: ?

**Comment by railsagainstignorance** [comment]
Looks fine to me as a work in progress.
- no list of mandatory env params in README
- no error checking for missing or invalid env params

File: bin/lib/homepage.js (hunk 2 above)
Comment (Lily2point0) [reply bin/lib/homepage.js]: ES6 notation

**Pull request merged**